openedx/openedx-platform
# MB-1192: [feat] Add course ended, not passing certificate status to dashboard messaging

This PR adds a message to a dashboard course card around a user who is in the "course ended, not passing" state. Now, when a user:
- Is in a verified, cert-bearing seat in a course
- the course has ended
- the user does not have a passing grade in the course

A new message will appear informing them that they have not passed, along with a button to go to their progress page and see grades.

## Testing instructions

- Create a verified seat in the demo course (or some other course). Enroll a user.
- fail the course
- change the end date to a date in the past
- check that the messaging properly appears on the dashboard

related themes PR: https://github.com/edx/edx-themes/pull/747

![image](https://user-images.githubusercontent.com/16495365/128394065-3a467e2a-6c8b-4ade-9476-9a2a76d395de.png)
